app/views/shared/_browse_related_topics.html.erb

```diff
@@ -1,15 +1,18 @@
-<% if page.related_topics.any? %>
+<% 
+  if page.related_topics.any? 
+  heading_text = t("second_level_browse.more_on_this_topic")
```

Comment:
I think you'll need to use a version of the `heading_text` that has it forced to use the English locale, e.g.

```Ruby
<% 
  if page.related_topics.any? 
  heading_text = t("second_level_browse.more_on_this_topic")
  ga4_heading_text = t("second_level_browse.more_on_this_topic", locale: :en)
%>
```

Reply:
Good spot, I guess there's no point in declaring a variable in that case